api/tests/Traits/CrawlerTrait.php

```diff
@@ -0,0 +1,732 @@
+<?php
+
+use Illuminate\Http\Request;
+use Symfony\Component\DomCrawler\Form;
+use Symfony\Component\DomCrawler\Crawler;
+use PHPUnit_Framework_ExpectationFailedException as PHPUnitException;
+
+/**
+ * With the future release of Lumen 5.1, this trait can most probably be
```

Comment:
@xiphiaz  This entire trait is now completely removed as of commit 9525d593. This trait is not needed anymore when Lumen is upgraded to 5.1.

Though I found a bug in 5.1.0 (That will more or less guaranteed be fixed in 5.1.1, next minor Lumen update) that forces us to override tearDown() in TestCase right now, otherwise mocking won't work. See commit fdac3f8c. So I'll add a @todo there.